app/views/monthlyContributions.scala.html

```diff
@@ -21,17 +21,24 @@
     <script type="text/javascript">
         window.guardian = window.guardian || {};
         window.guardian.uatMode = @uatMode;
-        window.guardian.user = {
-            id: "@user.id",
-            email: "@user.primaryEmailAddress",
-            @user.publicFields.displayName.map { displayName =>
-            displayName: "@displayName",
+        @user.map{ x =>
+            window.guardian.user = {
+                id: "@x.id"
+            ,
+            email: "@x.primaryEmailAddress"
+            ,
+            @x.publicFields.displayName.map { displayName =>
+                displayName: "@displayName"
+                ,
             }
-            @for(fields <- user.privateFields; firstName <- fields.firstName; lastName <- fields.secondName) {
-                firstName: "@firstName",
-                lastName: "@lastName",
+            @for(fields <- x.privateFields; firstName <- fields.firstName; lastName <- fields.secondName) {
+                firstName: "@firstName"
+                ,
+                lastName: "@lastName"
+                ,
```

Comment:
I may be missing something here but... has something happened to the formatting of these lines?